.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] Please make sure to delete the node_modules from the repository. Here is what you can do:
- Delete node modules, 
- Push changes 
- Run `npm install `